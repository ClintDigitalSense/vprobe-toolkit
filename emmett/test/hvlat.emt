HV_Exit {
   startTSC = TSC;
}

HV_Resume {
   if (startTSC) {
      ag[RIP, VMCB_exitCode] <- TSC - startTSC;
   }
}
