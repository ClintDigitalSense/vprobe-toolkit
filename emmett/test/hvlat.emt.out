(version 0.2)
(defaggr ag 1 0)
(definteger startTSC)

 ;;;;;
(vprobe HV_Exit ( do  ( setint startTSC   TSC)))
 ;;;;;
(vprobe HV_Resume ( do  ( cond  (startTSC( do  ( aggr  ag ( RIP) ()  (- TSC  startTSC )))))))